Commerce 9687 b

victoroscuro opened the pull request.

david-truong committed:
LPS-148816 add a preview for content and widget pages when reviewing publication changes.

jkappler committed:
LPS-154506 Respect empty values

Commit:
LPS-158121 Fix roles and permissions for Customer Portal

Commit:
LPS-158121 Add prompt login configuration for Site Initializer

andrea-ale-sbarra committed:
COMMERCE-9594 new placed order API (yaml file)

andrea-ale-sbarra committed:
COMMERCE-9594 new placed order API modules

andrea-ale-sbarra committed:
COMMERCE-9594 placed orders

andrea-ale-sbarra committed:
COMMERCE-9594 placed order items

andrea-ale-sbarra committed:
COMMERCE-9594 placed order comments

andrea-ale-sbarra committed:
COMMERCE-9594 placed order address

felipelorenz committed:
LPS-156875 search-experiences-test: Deflake SXPBlueprintSearchResultTest.testBoostContentsWithMoreVersions

https://issues.liferay.com/browse/LPS-156875

joseabelenda committed:
LPS-158121 add ConfigurationProvider

joseabelenda committed:
LPS-158121 save interface configuration

joseabelenda committed:
LPS-158289 implement saveGroupConfiguration(long groupId, String pid, Dictionary<String, Object> properties) throws ConfigurationException

joseabelenda committed:
LPS-158121 _addSiteSettings

joseabelenda committed:
LPS-158289 auto SF

joseabelenda committed:
LPS-158289 change testSaveGroupConfiguration to test using saveGroupConfiguration(long groupId, String pid, Dictionary<String, Object> properties)

joseabelenda committed:
LPS-158121 auto SF

joseabelenda committed:
LPS-158121 add portal-configuration-metatype-api dependency for test

joseabelenda committed:
LPS-158121 _assertSiteSettings

joseabelenda committed:
LPS-158121 add site-settings.json test resource

joseabelenda committed:
LPS-158121 remove unnecessary dependency

andrea-ale-sbarra committed:
COMMERCE-9458 placed order item shipments

andrea-ale-sbarra committed:
COMMERCE-9458 autogenerated code (gw buildREST)

joseabelenda committed:
LPS-158289 baseline

felipelorenz committed:
LPS-156875 search-experiences-test: Deflake SXPBlueprintSearchResultTest.testTextMatchOverMultipleFields_bestFields

https://issues.liferay.com/browse/LPS-156875

rijema committed:
Revert "LPS-155103 sort"

This reverts commit aa1dfe55ce046329ff64d8aba591f2a6b764e0c1.

rijema committed:
Revert "LPS-155103 Identifying upload as a object field"

This reverts commit 1ae87339d37fd49766ba876521747393b5b95ea3.

victoroscuro committed:
COMMERCE-9687 Fix failing testcases

AlmirFe committed:
LPS-156875 search-experiences-test: Print element's Score

AlmirFe committed:
LPS-156875 search-experiences-test: Deflake SXPBlueprintSearchResultTest.testTextMatchOverMultipleFields_mostFields

https://issues.liferay.com/browse/LPS-156875

Commit:
LPS-153687 TS config

Commit:
LPS-153687 Use ManageCollaborators component

Commit:
LPS-156875 search-experiences-test: Add Score per articles in AssertError message in SXPBlueprintSearchResultTest

Commit:
LPS-153687 Reorder files

Commit:
LPS-153687 FE unit testing

Commit:
LPS-153687 Adding styles and fix markup

Commit:
COMMERCE-9687 Adapt tests to Chrome 100 usage

Commit:
LPS-153687 Use the proper eror handling

Commit:
LPS-153687 Fix test onError

ling-alan-huang committed:
LPS-150378 SF, inline

victorg1991 committed:
LPS-158810 Remove FF from import/export

victorg1991 committed:
LPS-158810 Remove FF from page rendering

victorg1991 committed:
LPS-158810 Remove FF from page editor

victorg1991 committed:
LPS-158810 Remove FF from tests

victorg1991 committed:
LPS-158810 Remove FF from properties

ling-alan-huang committed:
LPS-150378 SF, inline

ling-alan-huang committed:
LPS-150378 SF, inline

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

RafaelCRC committed:
LPS-157176 (headless-admin-user-impl): new descriptiveName_i18n property in rest-openapi.yaml

RafaelCRC committed:
LPS-157176 (portal-impl): created getDescriptiveNameMap in GroupImpl

RafaelCRC committed:
LPS-157176 (headless-admin-user-impl): added descriptiveName_i18n _toSiteBrief

victoroscuro committed:
COMMERCE-9687 Fix new failing testcases

balazssk committed:
LPS-159106 If layoutStructureItem is null this assetListEntryUsage is not present in the current layoutStructure

Commit:
LPS-153350 Add validation to groovy script maximum number of lines available

AlmirFe committed:
LPS-156875 search-experiences-test: Add a setTitleMap and refactor the most fields test and the article Builder to make the test more readable

https://issues.liferay.com/browse/LPS-156875

p2kmgcl committed:
LPS-154506 Use isNullOrUndefined

p2kmgcl committed:
LPS-154506 Check original value

AliciaGarciaGarcia committed:
LPS-157505 rename edit_article to edit_kb_article

AliciaGarciaGarcia committed:
LPS-157505 rename edit_folder to edit_kb_folder

AliciaGarciaGarcia committed:
LPS-157505 rename edit_template to edit_kb_template

AliciaGarciaGarcia committed:
LPS-157505 rename view_article to view_kb_article

Commit:
LPS-159015 - ANNOUNCEMENTS

Commit:
COMMERCE-9458 SF

felipelorenz committed:
LPS-156875 search-experiences-test: SF

felipelorenz committed:
LPS-159041 search-experiences-rest-impl: Remove redundant dependecy

https://issues.liferay.com/browse/LPS-159041

seltong committed:
LPS-159060 Create External Reference Code field to Custom Objects

JessieZhao0714 committed:
LRQA-76248 Update locator of search icon

lfbesada committed:
LPS-158759 Add a class to assert AddFragmentEntryLinksMVCActionCommand

lfbesada committed:
LPS-158759 Add an integration test to assert that the fragmentEntryLinks contained by the fragmentComposition are added to the layout structure

lfbesada committed:
LPS-158759 SF, extract to a method to reuse

lfbesada committed:
LPS-158759 Add an integration test to specifically assert fragment compositions with fragmentEntryLinks that contain drop zone tags

lfbesada committed:
LPS-158759 Assert that the content returned by the JSON equals the result of rendering the fragmentEntryLink.

In the case of the fragments that contain a drop zone this assertion fails. That is because we are getting the JSON for a fragmentEntryLink using a layout structure that could be outdated. It could happen because, during the fragmentEntryLinkListeners invocations, the layout structure could be updated , as the DropZoneFragmentEntryLinkListener does.

So the updated render result of a fragmentEntryLink with a drop zone tag should have an "uuid" attribute that in the outdated ones is not present. That cause that the added assert fails.

We are fixing it with the following commits.

See:
1. Getting the JSON: https://github.com/liferay/liferay-portal/blob/8e2b7ff191e65471e9b436fb6c3726ed55bbbc93/modules/apps/layout/layout-content-page-editor-web/src/main/java/com/liferay/layout/content/page/editor/web/internal/portlet/action/AddFragmentEntryLinksMVCActionCommand.java#L145-L151
2. DropZoneFragmentEntryLinkListener updating the layout structure: https://github.com/liferay/liferay-portal/blob/67c4b5cc94e9985f57d7224c97ec1ae55ec50ce4/modules/apps/fragment/fragment-entry-processor/fragment-entry-processor-drop-zone/src/main/java/com/liferay/fragment/entry/processor/drop/zone/listener/DropZoneFragmentEntryLinkListener.java#L206-L225

lfbesada committed:
LPS-158759 Delay the fragmentEntryLink JSON calculation to after the listener's invocation

lfbesada committed:
LPS-158759 Assert that the layoutData returned by the JSON is updated.

In the case of the fragments that contain a drop zone this assertion fails. That is because we are using an outdated layoutStructure to get the JSON because the fragmentEntryLinkListeners are modifying the layoutStructure after we get it.

We are fixing it with the following commits

lfbesada committed:
LPS-158759 Ensure that all the JSON results are based on the updated LayoutStructure.

lfbesada committed:
LPS-158759 Apply to UpdateFormItemConfigMVCActionCommand, ensure that all the JSON results are based on the updated LayoutStructure

Commit:
LPS-158810 Modify macro to adapt new behavior

Commit:
LPS-153687 Rename prop

Commit:
LPS-153687 Add content-dashboard-web to SF  excludedDirNames

lfbesada committed:
LPS-159098 Add a specific feature flag

ealonso committed:
LPS-159116 Make it consistent with other places of the portal see LPS-146835

Commit:
LPS-153687 Fix mvc path

Commit:
LPS-153687 Fix prop type

victorg1991 committed:
LPS-153356 Create new page-content plugin and move the Page contents code there

victorg1991 committed:
LPS-153356 Add new panel

AliciaGarciaGarcia committed:
LPS-157505 fix route on test

victorg1991 committed:
LPS-153356 Auto SF

p2kmgcl committed:
LPS-85145 Add optional skip link

p2kmgcl committed:
LPS-85145 Add skip link to product menu button

p2kmgcl committed:
LPS-85145 Add skip link to simulation panel button

weller0246 committed:
LRQA-76240 Change locator to new one

weller0246 committed:
LRQA-76240 Add new locator for structure selection

p2kmgcl committed:
LPS-85145 buildLang

joseabelenda committed:
LPS-158121 no longer necessary

joseabelenda committed:
LPS-158121 add site settings

AliciaGarciaGarcia committed:
LPS-158794 id no user or no authenticated user, throw an exception

AliciaGarciaGarcia committed:
LPS-158794 check view permissions for user

rafaellajordao committed:
LPS-152906 Test Automation Add a share button to the info panel of the content dashboard

danvb112 committed:
LPS-153350 Add Language Key

victoroscuro committed:
COMMERCE-9687 Fix failing tests due to iframe issues and wrong clicks

danvb112 committed:
LPS-153350 Buildlang

seltong committed:
LPS-159060 Add feature flag to new external reference code field

victorg1991 committed:
LPS-153356 Adapt tests

joshuacords committed:
LPS-156875 search-experiences-test: just modify message before asserting

https://issues.liferay.com/browse/LPS-156875

ozysouza committed:
LPS-158810 Add macro go to ellpisis menu item in page structure

ozysouza committed:
LPS-158810 Add path to ellpisis icon in page structure

ozysouza committed:
LPS-158810 Delete unneeded macro and path

ozysouza committed:
LPS-158810 Update usages

brookedalton committed:
LPS-159142 Update how to select search scope

joseabelenda committed:
LPS-159015 is not necessary

joseabelenda committed:
LPS-159015 auto SF

RafaelCRC committed:
LPS-157176 generated build REST

RafaelCRC committed:
LPS-157176 generated build SERVICE

brookedalton committed:
LPS-159142 Add new path to select classic embedded search option

brookedalton committed:
LPS-159142 Update how to select search scope in ClassicSearch macro

calvinctvo committed:
LRAC-11569 Add documented pause macro for scrolling event

calvinctvo committed:
LRAC-11569 SF

Commit:
LPS-158810 Refactor locator

Commit:
LPS-158810 Refactor macro

Commit:
LPS-158810 Update usages

calvinctvo committed:
LRAC-11569 Change macro name

JanisZhang committed:
LPS-159047 org.Mockito.Matchers is removed in Mockito 4, use Mockito instead

brianchandotcom committed:
artifact:ignore fragment-impl 4.0.46 artifact properties

brianchandotcom committed:
artifact:ignore item-selector-upload-web 6.0.10 change log

brianchandotcom committed:
artifact:ignore item-selector-upload-web 6.0.10 prep next

brianchandotcom committed:
artifact:ignore item-selector-upload-web 6.0.10 artifact properties

brianchandotcom committed:
artifact:ignore item-selector-web 7.0.25 change log

brianchandotcom committed:
artifact:ignore item-selector-web 7.0.25 artifact properties

brianchandotcom committed:
artifact:ignore item-selector-web 7.0.25 prep next

brianchandotcom committed:
artifact:ignore journal-web 5.0.85 change log

brianchandotcom committed:
artifact:ignore journal-web 5.0.85 artifact properties

brianchandotcom committed:
artifact:ignore journal-web 5.0.85 prep next

brianchandotcom committed:
artifact:ignore knowledge-base-web 5.0.46 change log

brianchandotcom committed:
artifact:ignore knowledge-base-web 5.0.46 artifact properties

brianchandotcom committed:
artifact:ignore knowledge-base-web 5.0.46 prep next

brianchandotcom committed:
artifact:ignore layout-admin-web 5.0.89 change log

brianchandotcom committed:
artifact:ignore layout-admin-web 5.0.89 prep next

brianchandotcom committed:
artifact:ignore layout-admin-web 5.0.89 artifact properties

brianchandotcom committed:
artifact:ignore layout-content-page-editor-web 3.0.111 change log

brianchandotcom committed:
artifact:ignore layout-content-page-editor-web 3.0.111 artifact properties

brianchandotcom committed:
artifact:ignore layout-content-page-editor-web 3.0.111 prep next

brianchandotcom committed:
artifact:ignore object-web 1.0.98 change log

brianchandotcom committed:
artifact:ignore object-web 1.0.98 prep next

brianchandotcom committed:
artifact:ignore object-web 1.0.98 artifact properties

brianchandotcom committed:
artifact:ignore portal-search-web 6.0.59 change log

brianchandotcom committed:
artifact:ignore portal-search-web 6.0.59 artifact properties

brianchandotcom committed:
artifact:ignore portal-search-web 6.0.59 prep next

brianchandotcom committed:
artifact:ignore product-navigation-product-menu-web 6.0.51 change log

brianchandotcom committed:
artifact:ignore product-navigation-product-menu-web 6.0.51 artifact properties

brianchandotcom committed:
artifact:ignore product-navigation-product-menu-web 6.0.51 prep next

brianchandotcom committed:
artifact:ignore sharing-web 3.0.32 change log

brianchandotcom committed:
artifact:ignore sharing-web 3.0.32 prep next

brianchandotcom committed:
artifact:ignore sharing-web 3.0.32 artifact properties

brianchandotcom committed:
artifact:ignore site-navigation-item-selector-web 4.0.28 change log

brianchandotcom committed:
artifact:ignore site-navigation-item-selector-web 4.0.28 prep next

brianchandotcom committed:
artifact:ignore site-navigation-item-selector-web 4.0.28 artifact properties

brianchandotcom committed:
artifact:ignore users-admin-web 6.0.55 change log

brianchandotcom committed:
artifact:ignore users-admin-web 6.0.55 prep next

brianchandotcom committed:
artifact:ignore users-admin-web 6.0.55 artifact properties

brianchandotcom committed:
artifact:ignore wiki-web 7.0.52 change log

brianchandotcom committed:
artifact:ignore wiki-web 7.0.52 prep next

brianchandotcom committed:
artifact:ignore wiki-web 7.0.52 artifact properties

brianchandotcom committed:
artifact:ignore journal-content-asset-addon-entry-related-assets 6.0.8 change log

brianchandotcom committed:
artifact:ignore journal-content-asset-addon-entry-related-assets 6.0.8 prep next

brianchandotcom committed:
artifact:ignore journal-content-asset-addon-entry-related-assets 6.0.8 artifact properties

brianchandotcom committed:
artifact:ignore layout-type-controller-content 4.0.23 change log

brianchandotcom committed:
artifact:ignore layout-type-controller-content 4.0.23 prep next

brianchandotcom committed:
artifact:ignore layout-type-controller-content 4.0.23 artifact properties

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
artifact:ignore commerce-product-item-selector-web 4.0.22 change log

brianchandotcom committed:
artifact:ignore commerce-product-item-selector-web 4.0.22 prep next

brianchandotcom committed:
artifact:ignore commerce-product-item-selector-web 4.0.22 artifact properties

brianchandotcom committed:
artifact:ignore commerce-product-type-grouped-web 4.0.29 change log

brianchandotcom committed:
artifact:ignore commerce-product-type-grouped-web 4.0.29 prep next

brianchandotcom committed:
artifact:ignore commerce-product-type-grouped-web 4.0.29 artifact properties

brianchandotcom committed:
artifact:ignore commerce-warehouse-web 4.0.28 change log

brianchandotcom committed:
artifact:ignore commerce-warehouse-web 4.0.28 artifact properties

brianchandotcom committed:
artifact:ignore commerce-warehouse-web 4.0.28 prep next

brianchandotcom committed:
artifact:ignore journal-content-web 6.0.31 change log

brianchandotcom committed:
artifact:ignore journal-content-web 6.0.31 prep next

brianchandotcom committed:
artifact:ignore journal-content-web 6.0.31 artifact properties

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
artifact:ignore analytics-reports-web 2.0.54 change log

brianchandotcom committed:
artifact:ignore analytics-reports-web 2.0.54 artifact properties

brianchandotcom committed:
artifact:ignore analytics-reports-web 2.0.54 prep next

brianchandotcom committed:
artifact:ignore document-library-opener-onedrive-web 3.0.27 change log

brianchandotcom committed:
artifact:ignore document-library-opener-onedrive-web 3.0.27 prep next

brianchandotcom committed:
artifact:ignore document-library-opener-onedrive-web 3.0.27 artifact properties

brianchandotcom committed:
artifact:ignore enterprise-product-notification-web 1.0.7 change log

brianchandotcom committed:
artifact:ignore enterprise-product-notification-web 1.0.7 prep next

brianchandotcom committed:
artifact:ignore enterprise-product-notification-web 1.0.7 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-designer-web 5.0.69 change log

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-designer-web 5.0.69 artifact properties

brianchandotcom committed:
artifact:ignore portal-workflow-kaleo-designer-web 5.0.69 prep next

brianchandotcom committed:
artifact:ignore saml-web 5.0.34 change log

brianchandotcom committed:
artifact:ignore saml-web 5.0.34 prep next

brianchandotcom committed:
artifact:ignore saml-web 5.0.34 artifact properties

brianchandotcom committed:
artifact:ignore search-experiences-web 3.0.11 change log

brianchandotcom committed:
artifact:ignore search-experiences-web 3.0.11 prep next

brianchandotcom committed:
artifact:ignore search-experiences-web 3.0.11 artifact properties

brianchandotcom committed:
artifact:ignore segments-experiment-web 3.0.42 change log

brianchandotcom committed:
artifact:ignore segments-experiment-web 3.0.42 prep next

brianchandotcom committed:
artifact:ignore segments-experiment-web 3.0.42 artifact properties

brianchandotcom committed:
build.gradle auto SF

brianchandotcom committed:
7.4.3.36 GA36 Release Apps: account, adaptive-media, address, alloy, analytics, announcements, application-list, asset, batch-engine, batch-planner, bean-portlet, blogs, bulk, calendar, captcha, change-tracking, changeset, click-to-chat, client-extension, comment, commerce, configuration-admin, connected-app, contacts, content-dashboard, cookies, data-cleanup, data-engine, depot, diff, digital-signature, dispatch, document-library-opener, document-library, dynamic-data-lists, dynamic-data-mapping, expando, export-import, external-reference, flags, fragment, friendly-url, frontend-css, frontend-data-set, frontend-editor, frontend-icons, frontend-js, frontend-taglib, frontend-theme, frontend-token, gogo-shell, google-places, headless, iframe, image-uploader, image, imageio-plugins, info, invitation, item-selector, journal, json-storage, knowledge-base, layout, learn, license-manager, list-type, login, mail, map, mentions, message-boards, microsoft-translator, monitoring, nested-portlets, normalizer, notification, notifications, oauth-client, oauth2-provider, object, organizations, password-policies-admin, petra, plugins-admin, portal-background-task, portal-cache, portal-configuration, portal-instances, portal-language, portal-lock, portal-odata, portal-osgi-debug, portal-portlet-bridge, portal-relationship, portal-remote, portal-reports-engine, portal-rules-engine, portal-scheduler, portal-scripting, portal-search-elasticsearch7, portal-search, portal-security-audit, portal-security-sso, portal-security, portal-settings, portal-store, portal-template, portal-url-builder, portal-validation, portal-vulcan, portal-workflow, portal, portlet-configuration, portlet-dependency-factory, portlet-display-template, product-navigation, questions, ratings, reading-time, redirect, release-feature-flag, roles, rss, segments, server, sharing, site-initializer, site-navigation, site, social, staging, style-book, subscription, template, text-localizer, translation, trash, upload, user-associated-data, user-groups-admin, users-admin, view-count, websocket, wiki, words, xstream

brianchandotcom committed:
7.4.13 Update 36 Release Apps: account, adaptive-media, address, alloy, analytics, announcements, application-list, asset, batch-engine, batch-planner, bean-portlet, blogs, bulk, calendar, captcha, change-tracking, changeset, click-to-chat, client-extension, comment, commerce, configuration-admin, connected-app, contacts, content-dashboard, cookies, data-cleanup, data-engine, depot, diff, digital-signature, dispatch, document-library-opener, document-library, dynamic-data-lists, dynamic-data-mapping, expando, export-import, external-reference, flags, fragment, friendly-url, frontend-css, frontend-data-set, frontend-editor, frontend-icons, frontend-js, frontend-taglib, frontend-theme, frontend-token, gogo-shell, google-places, headless, iframe, image-uploader, image, imageio-plugins, info, invitation, item-selector, journal, json-storage, knowledge-base, layout, learn, license-manager, list-type, login, mail, map, mentions, message-boards, microsoft-translator, monitoring, nested-portlets, normalizer, notification, notifications, oauth-client, oauth2-provider, object, organizations, password-policies-admin, petra, plugins-admin, portal-background-task, portal-cache, portal-configuration, portal-instances, portal-language, portal-lock, portal-odata, portal-osgi-debug, portal-portlet-bridge, portal-relationship, portal-remote, portal-reports-engine, portal-rules-engine, portal-scheduler, portal-scripting, portal-search-elasticsearch7, portal-search, portal-security-audit, portal-security-sso, portal-security, portal-settings, portal-store, portal-template, portal-url-builder, portal-validation, portal-vulcan, portal-workflow, portal, portlet-configuration, portlet-dependency-factory, portlet-display-template, product-navigation, questions, ratings, reading-time, redirect, release-feature-flag, roles, rss, segments, server, sharing, site-initializer, site-navigation, site, social, staging, style-book, subscription, template, text-localizer, translation, trash, upload, user-associated-data, user-groups-admin, users-admin, view-count, websocket, wiki, words, xstream, antivirus, enterprise-product-notification, multi-factor-authentication, portal-search-elasticsearch-cross-cluster-replication, portal-search-elasticsearch-monitoring, portal-search-learning-to-rank, portal-search-similar-results, portal-search-tuning, saml, search-experiences

lfbesada committed:
LPS-158374 SF Auto

AliciaGarciaGarcia committed:
LPS-158880 if no view permission return success false

AliciaGarciaGarcia committed:
LPS-158880 prefer reference instead util

AliciaGarciaGarcia committed:
LPS-158880 if call is from GET request

Commit:
LPS-158730 Modify function to execute only once

Commit:
LPS-158730 Refactor case

p2kmgcl committed:
LPS-159190 Replace aria-label with title

brianchandotcom committed:
Prepare for U37

brianchandotcom committed:
LPS-158810 SF

liferay-translation committed:
LPS-77699 Update Translations

Commit:
COMMERCE-9736 batch-engine-service - update tests to reflect apache-commons csvPrinter behavior (/r/n record separator and first column empty string is always escaped)

Commit:
COMMERCE-9736 batch-engine-service - unnecessary

magjed4289 committed:
LPS-158454 Ignore test cases till the acceptance criteria re clear

magjed4289 committed:
LPS-158454 SF

balazssk committed:
LPS-159106 Forward segmentsExperienceId instead of trying to read it again

kellylisbino committed:
LPS-158275 Update DDMTemplate

kellylisbino committed:
LPS-159224 Update LayoutPageTemplate

brianchandotcom committed:
Revert "LPS-157670 LPP-41834 Use memory"

This reverts commit 4176edca7ea9574e63805addede8d128ba6abdf2.

brianchandotcom committed:
LPS-157670 LPP-41834 Use pattern from com.liferay.portal.upgrade.v7_0_5.UpgradeExpando#deleteOrphanExpandoRow because of https://explainextended.com/2009/09/16/not-in-vs-not-exists-vs-left-join-is-null-postgresql/

brianchandotcom committed:
LPS-148816 Use concat

kellylisbino committed:
LPS-159229 Update Resource Permissions

brianchandotcom committed:
LPS-157925 Add helper method for tests

brianchandotcom committed:
LPS-157925 Regen

brianchandotcom committed:
LPS-157925 Add test

brianchandotcom committed:
LPS-159015 We are returning a map

brianchandotcom committed:
LPS-159015 Not needed

brianchandotcom committed:
LPS-159015 The entity is not DDMStructureEntry, but just DDMStructure

brianchandotcom committed:
LPS-159015 Rename

Commit:
COMMERCE-9736 batch-engine-service - csvParser parses string representation of array so that every element is a new column value

Commit:
LPS-159237 Remove user-accounts.json and user-roles.json from site-initializer

brianchandotcom committed:
LPS-158121 Inline

brianchandotcom committed:
LPS-158121 No benefit

brianchandotcom committed:
LPS-158121 Inline

brianchandotcom committed:
Baseline

brianchandotcom committed:
LPS-156875 Use concat

Commit:
COMMERCE-9736 batch-engine-service - use apache-commons csv writer to generate csv export file (fix unescaped column values)

drewbrokke committed:
LPS-158933 roles-admin-web: escapes role name for JS

Luizcarlosqueiroz committed:
LRQA-76265 Change order of assets asserts

brianchandotcom committed:
COMMERCE-9736 SF

joshchong committed:
LPS-159270 Deactivate contributor that is incorrectly named

brianchandotcom committed:
LPS-151592 This is true for 6.1.x as well and the fix in 012914c08a81a6548dabf78642f4f6391e8a30f3 was not complete

brianchandotcom committed:
LPS-147889 This exposed a bug in LPS-68555 at 9ed5e7950b9911c63c244e27129580b621924677 where there was a typo

Commit:
LPS-159278 Remove pause from site macros

Commit:
LPS-159279 Remove pause from site navigation macros
